### PR TITLE
(GH-80) Package Id too long and no dashes Note

### PR DESCRIPTION
**Pull request by gep13**
Relates to #80

### CODE REVIEW

**Comment by gep13** [comment]
@ferventcoder first of a few, but need to head out just now.

Let me know what you think.

File: src/chocolatey.package.validator/infrastructure.app/rules/PackageIdTooLongWithNoDashesNote.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                return
+@"The package id is too long and should contain '-' to make it more readable. The reviewer will suggest a change to the package id. [More...](https://github.com/chocolatey/package-validator/wiki/PackageIdTooLongWithNoDashes)";
+            }
+        }
+
+        public override PackageValidationOutput is_valid(IPackage package)
+        {
+            var packageId = package.Id.to_lower();
+
+            if (packageId.Length < 25) return true;
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/PackageIdTooLongWithNoDashesNote.cs]: Good first pass - Wondering if the length being over 10-15 is more appropriate now though.

Comment (gep13) [inline src/chocolatey.package.validator/infrastructure.app/rules/PackageIdTooLongWithNoDashesNote.cs]: @ferventcoder  15 sounds like a good number to me

Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/PackageIdTooLongWithNoDashesNote.cs]: :+1:

**Comment by gep13** [comment]
@ferventcoder changed to 15.

File: src/chocolatey.package.validator/infrastructure.app/rules/PackageIdTooLongWithNoDashesNote.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    using infrastructure.rules;
+
+    public class PackageIdTooLongWithNoDashesNote : BasePackageRule
+    {
+        public override string ValidationFailureMessage
+        {
+            get
+            {
+                return
+@"The package id is too long and should contain '-' to make it more readable. The reviewer will suggest a change to the package id. [More...](https://github.com/chocolatey/package-validator/wiki/PackageIdTooLongWithNoDashes)";
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/PackageIdTooLongWithNoDashesNote.cs]: If this is a brand new package that has never went through approval before, the reviewer will suggest a change to the package id.

**Comment by gep13** [comment]
@ferventcoder corrected

**Pull request merged**